scopes/component/component-filters/envs-filter.module.scss

```diff
@@ -40,15 +40,53 @@
   width: 100%;
 }
 
+.envDropdownItemContainer {
+  display: flex;
+  width: calc(100% - 24px);
+  align-items: baseline;
+  flex-direction: column;
+}
+
+.envDropdownItemIconContainer {
+  display: flex;
+  height: 16px;
+  width: 16px;
+}
+
 .envDropdownItem {
   padding-left: 8px;
   display: flex;
   justify-content: space-between;
   width: calc(100% - 24px);
+  font-size: var(--bit-p-md, 16px);
 }
 
 .envDropdownItemLabel,
 .envDropdownItemIcon {
   display: flex;
   justify-content: space-between;
+  padding-left: 8px;
+}
+.description {
+  font-size: var(--bit-p-xs, 14px);
+  padding-left: 8px;
+  padding-right: 16px;
+  color: var(--bit-text-color-light, #6c707c);
+  width: calc(100% - 24px);
```

Comment:
check if you can move the width to the top most divs. will prevent using in each class